sxs/utilities/lvcnr/waveforms.py

```diff
@@ -2,7 +2,8 @@
 
 import numpy as np
 import h5py
-
+import sxs
+import spherical_functions as sf
```

Comment:
Instead of `sf.LM_index(L, M, h.ell_min)`, you should just use `h.index(L, M)`, so you shouldn't need such an import at all.  (But also, `spherical_functions` is the C-style dependency; we need to stick to `spherical` for `sxs.)

```suggestion
```